src/Error/Renderer/WebExceptionRenderer.php

```diff
@@ -425,7 +425,7 @@ protected function _outputMessage(string $template, bool $skipControllerCheck =
                 $this->controller->viewBuilder()->setTemplate($method);
 
                 $reflectionMethod = new ReflectionMethod($this->controller, $method);
-                $result = $reflectionMethod->invoke($this->controller);
+                $result = $reflectionMethod->invoke($this->controller, $this->error);
```

Comment:
I had to match the code to the e.g. @markstory added to the docs https://github.com/cakephp/docs/blob/5.next/en/development/errors.rst?plain=1#L228 :laughing:

Reply:
I wanted to keep this methods protected so that they can't be called through URLs like `/error/foo`.

Reply:
https://github.com/cakephp/docs/pull/7925

Reply:
We don't have to match what I put in the docs 🤣 The `ExceptionRenderer` layer methods also get the exception passed in, and I thought it would be good to have consistency with how we're doing custom exceptions there.

Reply:
I was just joking :)   Passing the exception as an argument is a good idea.